Makefile

```diff
@@ -33,14 +33,23 @@ endif
 
 verify-crds:
 ifneq ($(BUILD_IN_CONTAINER),1)
-	./hack/verify-crds.sh
+	./hack/verify-crds.sh crds
+	./hack/verify-crds.sh crds-v1beta1
 endif
 
 update-codegen:
 	./hack/update-codegen.sh
 
 update-crds:
-	go run sigs.k8s.io/controller-tools/cmd/controller-gen schemapatch:manifests=./deployments/common/ paths=./pkg/apis/configuration/... output:dir=./deployments/common
+	go run sigs.k8s.io/controller-tools/cmd/controller-gen schemapatch:manifests=./deployments/common/crds/ paths=./pkg/apis/configuration/... output:dir=./deployments/common/crds
+	go run sigs.k8s.io/controller-tools/cmd/controller-gen schemapatch:manifests=./deployments/common/crds-v1beta1/ paths=./pkg/apis/configuration/... output:dir=./deployments/common/crds-v1beta1
+	@make update-crds-helm
+
+update-crds-helm:
+	@for f in deployments/common/crds-v1beta1/*.yaml; do \
+		echo "updating $$(basename $$f) crd for helm"; \
+		kubectl label -f $$f --local=true -o yaml app.kubernetes.io/name=nginx-ingress > deployments/helm-chart/crds/$$(basename $$f); \
+	done
```

Comment:
Just so you know, these labels are not essential. For the case of simplicity, I think you can remove this Makefile target if you want

```suggestion
```